File: src/test/java/me/laus/tallinn/transport/util/FileSystemUtilsTest.java
package me.laus.tallinn.transport.util;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class FileSystemUtilsTest {
    @Test
    void stopFileIsNotNull() {
        assertNotNull(FileSystemUtils.DEFAULT_STOPS_PATH);
    }
}

File: src/test/java/me/laus/tallinn/transport/util/FileSystemUtilsTest.kt
package me.laus.tallinn.transport.util

import org.junit.jupiter.api.Assertions
import me.laus.tallinn.transport.util.FileSystemUtils
import org.junit.jupiter.api.Test

internal class FileSystemUtilsTest {
    @Test
    fun stopFileIsNotNull() {
        Assertions.assertNotNull(FileSystemUtils.DEFAULT_STOPS_PATH)
    }
}